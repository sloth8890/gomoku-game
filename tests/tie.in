place A1
place B1
place A2
place B2
place A3
place B3
place C1
place D1
place C2
place D2
place C3
place D3
place E1
place F1
place E2
place F2
place E3
place F3
place G1
place H1
place G2
place H2
place G3
place H3
place I1
place J1
place I2
place J2
place I3
place J3
place K1
place L1
place K2
place L2
place K3
place L3
place M1
place N1
place M2
place N2
place M3
place N3
place O1
place P1
place O2
place P2
place O3
place P3
place Q1
place R1
place Q2
place R2
place Q3
place R3
place B4
place A4
place B5
place A5
place B6
place A6
place B7
place A7
place D4
place C4
place D5
place C5
place D6
place C6
place D7
place C7
place F4
place E4
place F5
place E5
place F6
place E6
place F7
place E7
place H4
place G4
place H5
place G5
place H6
place G6
place H7
place G7
place J4
place I4
place J5
place I5
place J6
place I6
place J7
place I7
place L4
place K4
place L5
place K5
place L6
place K6
place L7
place K7
place N4
place M4
place N5
place M5
place N6
place M6
place N7
place M7
place P4
place O4
place P5
place O5
place P6
place O6
place P7
place O7
place R4
place Q4
place R5
place Q5
place R6
place Q6
place R7
place Q7
place A8
place B8
place A9
place B9
place A10
place B10
place A11
place B11
place C8
place D8
place C9
place D9
place C10
place D10
place C11
place D11
place E8
place F8
place E9
place F9
place E10
place F10
place E11
place F11
place G8
place H8
place G9
place H9
place G10
place H10
place G11
place H11
place I8
place J8
place I9
place J9
place I10
place J10
place I11
place J11
place K8
place L8
place K9
place L9
place K10
place L10
place K11
place L11
place M8
place N8
place M9
place N9
place M10
place N10
place M11
place N11
place O8
place P8
place O9
place P9
place O10
place P10
place O11
place P11
place Q8
place R8
place Q9
place R9
place Q10
place R10
place Q11
place R11
place B12
place A12
place B13
place A13
place B14
place A14
place B15
place A15
place D12
place C12
place D13
place C13
place D14
place C14
place D15
place C15
place F12
place E12
place F13
place E13
place F14
place E14
place F15
place E15
place H12
place G12
place H13
place G13
place H14
place G14
place H15
place G15
place J12
place I12
place J13
place I13
place J14
place I14
place J15
place I15
place L12
place K12
place L13
place K13
place L14
place K14
place L15
place K15
place N12
place M12
place N13
place M13
place N14
place M14
place N15
place M15
place P12
place O12
place P13
place O13
place P14
place O14
place P15
place O15
place R12
place Q12
place R13
place Q13
place R14
place Q14
place R15
place Q15
place A16
place B16
place A17
place B17
place A18
place B18
place A19
place B19
place C16
place D16
place C17
place D17
place C18
place D18
place C19
place D19
place E16
place F16
place E17
place F17
place E18
place F18
place E19
place F19
place G16
place H16
place G17
place H17
place G18
place H18
place G19
place H19
place I16
place J16
place I17
place J17
place I18
place J18
place I19
place J19
place K16
place L16
place K17
place L17
place K18
place L18
place K19
place L19
place M16
place N16
place M17
place N17
place M18
place N18
place M19
place N19
place O16
place P16
place O17
place P17
place O18
place P18
place O19
place P19
place Q16
place R16
place Q17
place R17
place Q18
place R18
place Q19
place R19
place S19
place S15
place S18
place S14
place S17
place S13
place S16
place S12
place S11
place S7
place S10
place S6
place S9
place S5
place S8
place S4
place S3
place S2
place S1
