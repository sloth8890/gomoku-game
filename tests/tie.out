Wow, a tie!
A1B1A2B2A3B3C1D1C2D2C3D3E1F1E2F2E3F3G1H1G2H2G3H3I1J1I2J2I3J3K1L1K2L2K3L3M1N1M2N2M3N3O1P1O2P2O3P3Q1R1Q2R2Q3R3B4A4B5A5B6A6B7A7D4C4D5C5D6C6D7C7F4E4F5E5F6E6F7E7H4G4H5G5H6G6H7G7J4I4J5I5J6I6J7I7L4K4L5K5L6K6L7K7N4M4N5M5N6M6N7M7P4O4P5O5P6O6P7O7R4Q4R5Q5R6Q6R7Q7A8B8A9B9A10B10A11B11C8D8C9D9C10D10C11D11E8F8E9F9E10F10E11F11G8H8G9H9G10H10G11H11I8J8I9J9I10J10I11J11K8L8K9L9K10L10K11L11M8N8M9N9M10N10M11N11O8P8O9P9O10P10O11P11Q8R8Q9R9Q10R10Q11R11B12A12B13A13B14A14B15A15D12C12D13C13D14C14D15C15F12E12F13E13F14E14F15E15H12G12H13G13H14G14H15G15J12I12J13I13J14I14J15I15L12K12L13K13L14K14L15K15N12M12N13M13N14M14N15M15P12O12P13O13P14O14P15O15R12Q12R13Q13R14Q14R15Q15A16B16A17B17A18B18A19B19C16D16C17D17C18D18C19D19E16F16E17F17E18F18E19F19G16H16G17H17G18H18G19H19I16J16I17J17I18J18I19J19K16L16K17L17K18L18K19L19M16N16M17N17M18N18M19N19O16P16O17P17O18P18O19P19Q16R16Q17R17Q18R18Q19R19S19S15S18S14S17S13S16S12S11S7S10S6S9S5S8S4S3S2S1
Thank you for playing!